cf/i18n/resources/es-es.all.json

```diff
@@ -2028,8 +2028,8 @@
     "translation": "Error al captar paquetes de compilación.\n{{.Error}}"
   },
   {
-    "id": "Failed fetching domains for organization {{.OrgName}}.\n{{.Err}}",
-    "translation": "Error al captar dominios para la organización {{.OrgName}}.\n{{.Err}}"
+    "id": "Failed fetching domains for organization %s.\n{{.Err}}",
```

Comment:
Why are you changing the message id? The code seems to use [{{.OrgName}}](https://github.com/cloudfoundry/cli/blob/master/cf/commands/route/routes.go#L96).